flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -238,9 +242,22 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 *        written. This is to ensure at least the name of the {@link TableSink} is provided.
 	 * @param sinkPathContinued The remaining part of the path of the registered {@link TableSink} to which the
 	 *        {@link Table} is written.
+	 * @deprecated use {@link #insertInto(String, Table)}
 	 */
+	@Deprecated
 	void insertInto(Table table, String sinkPath, String... sinkPathContinued);
 
+	/**
+	 * Writes the {@link Table} to a {@link TableSink} that was registered under the specified name.
```

Comment:
I would not mention `TableSink` or `sink` here anymore. Maybe:

`Instructs to write the content of a {@link Table} API object into a table.`